pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -28,9 +26,12 @@ let
 
 in makeScopeWithSplicing' {
   otherSplices = generateSplicesForMkScope "netbsd";
-  f = (self: let
-    inherit (self) mkDerivation;
-  in {
+  f = (self: lib.packagesFromDirectoryRecursive {
+    callPackage = self.callPackage;
+    directory = ./pkgs;
+  } // (let inherit (self) mkDerivation; in {
+
+  inherit defaultMakeFlags fetchNetBSD netbsdSetupHook;
```

Comment:
Can we move these 3 from the let block above directly into the inherit? I think nothing else is using them / if things are, we can get them via `self.<name>` instead? we can remove the `let..in` then.

Reply:
Also now that we basically have all-new and shorter content within this `{....}` we should indent two more spaces to fix the indentation that was broken :).

Reply:
Finally `netbsdSetupHook` can be a package, see `freebsdSetupHook` which is done with a `package.nix` and shell script file,

Reply:
done